src/Pages/UserPages/Landing/Landing.tsx

```diff
@@ -1,19 +1,292 @@
+    try {
+      // @ts-ignore
+      const element = await dispatch(RemoveFavoriteItem(roomId));
+      console.log(element)
+      // @ts-ignore
+      toast.success(element?.payload?.message, {
+        autoClose: 2000,
+        theme: "colored",
+      });
+    } catch (error) {
+      // toast.error("Error fetching data:", error);
+    }
+  }, [dispatch]);
+
+
+
+  const open = Boolean(anchorEl);
+
+
   return <>
-    <div>Landing</div>
+    <Box component="section" className="landingSec">
+
+      <Box className="leftCon">
+
+        <Typography variant="h1" className='title'>Forget Busy Work, Start Next Vacation</Typography>
+        <Typography className='subTitle'>We provide what you need to enjoy your holiday with family. Time to make another memorable moments.</Typography>
+
+        <Box className="bookingCon">
+
+          <Typography variant="h3" className='bookingTitle'>Start Booking</Typography>
+          <Typography variant="h4" className='subBookingTitle'>Pick a Date</Typography>
+
+
+
+        </Box>
+
+        <Box className="exploreCon">
+          <Button className="caleBtn" onClick={handleButtonClick} variant="contained" color="primary">
+            <CalendarMonth />
+          </Button>
+
+          <Popover
+            open={open}
+            anchorEl={anchorEl}
+            onClose={handlePopoverClose}
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'center',
+            }}
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'center',
+            }}
+          >
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DemoContainer components={['DateRangeCalendar']}>
+                <DateRangeCalendar
+                  value={selectedDateRange}
+                  onChange={handleCalendarChange}
+                  onAccept={handleDateSelected}
+                />
+              </DemoContainer>
+            </LocalizationProvider>
+          </Popover>
+
+
+          <TextField
+            className='calendarField'
+            label="Selected Date Range"
+            value={`${selectedDateRange[0]?.format('YYYY-MM-DD')} - ${selectedDateRange[1]?.format('YYYY-MM-DD')}`}
+          />
+
+          <Box className="capacityCon">
+            <Button onClick={handleIncrease} className="caleBtn" variant="contained" color="primary">
+              <Add />
+            </Button>
+            <TextField
+              className='calendarField'
+              label="Capacity"
+              value={`${personsCount} person`}
+            />
+            <Button onClick={handleDecrease} className="caleBtn" variant="contained" color="error">
+              <Remove />
+            </Button>
+          </Box>
+
+        </Box>
+
+        <Button className="submitExplore" onClick={()=>navigate(`/explore/startDate=${startDate}/endDate=${endDate}/persons=${personsCount}`)} variant="contained" color="primary">
```

Comment:
This path indicates that we have three different paths, startDate, endDate, and another one called persons=personsCount.
We can store these values in the router history, and when moving to the explore page, we can set these values in the searchParams.